kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/AlCustomizationSettings.java

```diff
@@ -587,6 +588,14 @@ public void setFilterGallery(Map<String, Boolean> filterGallery) {
         this.filterGallery = filterGallery;
     }
 
+    public boolean isEnableShareConversation() {
+        return enableShareConversation;
```

Comment:
@reytum What can we share by enabling it? An attachment? 
A description would be great!

Reply:
This is for internal use only, for Agent app, to share the conversation url